.vscode/tasks.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Since, the default features would be empty, building the crate to test is now useless, and shall be omitted